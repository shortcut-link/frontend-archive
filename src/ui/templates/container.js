import styled from 'styled-components';

import { WithTag } from 'lib/styled-components';

export const Container = styled(WithTag)`
  display: inherit;

  max-width: 65rem;
  width: 100%;
  padding: 1rem;
`;
