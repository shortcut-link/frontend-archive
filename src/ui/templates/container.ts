import styled from 'styled-components';

import { WithTag, mixins } from 'lib/styled-components';

export const Container = styled(WithTag)`
  ${mixins}
  display: inherit;

  max-width: 50rem;
  width: 100%;
  padding: 1rem;
`;
