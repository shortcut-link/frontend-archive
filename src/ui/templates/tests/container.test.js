import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Container } from '../container';

it('should render CSS container', () => {
  const tree = renderer.create(<Container />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule('max-width', '50rem');
});
