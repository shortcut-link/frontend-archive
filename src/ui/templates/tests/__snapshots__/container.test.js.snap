// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CSS container 1`] = `
.c0 {
  display: inherit;
  max-width: 50rem;
  width: 100%;
  padding: 1rem;
}

<div
  className="c0"
/>
`;
